libs.elimpl/README

```diff
@@ -1,4 +1,21 @@
-The EL impl library in external/el-impl.jar is patched with file from the 'https://netbeans.org/bugzilla/show_bug.cgi?id=271419s'.
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+The EL impl library in external/el-impl.jar is patched with file from the 'https://netbeans.org/bugzilla/show_bug.cgi?id=271419'.
```

Comment:
This is a readme, I would use the short version of the header:
https://www.apache.org/legal/src-headers.html#is-a-short-form-of-the-source-header-available

Reply:
Fixed.